source/ed/ed_persistor.cpp

```diff
@@ -130,7 +130,8 @@ void EdPersistor::insert_admins(const ed::Georef& data){
     for(const auto& itm : data.admins){
         if(itm.second->is_used){
             this->lotus.insert({std::to_string(itm.second->id), itm.second->name, itm.second->insee,
-                               itm.second->level, this->to_geographic_point(itm.second->coord),"admin:" + itm.second->insee});
+                               itm.second->level, this->to_geographic_point(itm.second->coord),
```

Comment:
I find it strange to do stuff here.

you are sure it is needed ? why isn't it done in osm2ed ?